lib/friendly_shipping/services/ups_freight/shipping_methods.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module FriendlyShipping
+  module Services
+    class UpsFreight
```

Comment:
`UpsFreight` is a `class`, there is no `module UpsFreight`. This is a gem, so Rails' strict naming conventions don't apply, and the thing is that all we need is a constant containing an Array. I think this can stay as-is.